app/containers/MusicContainer/tests/index.test.js

```diff
@@ -29,6 +29,10 @@ describe('<MusicContainer /> tests', () => {
     const { getByTestId } = render(<IntlProvider locale="en"><MusicContainer /></IntlProvider>)
     expect(getByTestId('search-bar')).toBeTruthy;
   })
+  it('should render grid', () => {
+    const { getByTestId } = render(<IntlProvider locale="en"><MusicContainer /></IntlProvider>)
```

Comment:
this doesn't seem right. Though we don't have any tracks passed as prop, the grid is still rendered. 

We should have a default state when there are no tracks to be rendered